code/sorting/src/cocktail_sort/cocktail_sort.cpp

```diff
@@ -0,0 +1,59 @@
+#include <stdio.h> 
+#include <string.h> 
+#include<bits/stdc++.h>
```

Comment:
@rudrakshi99 I believe someone else is already working on this issue at the moment. The code in #4889 is more conformant to the style of this repository so I will likely merge that instead of this. Consider working on a different issue.

Reply:
ok sir

Reply:
@arnavb  Sir, will u please review my other PRs as they are not merged yet.